src/components/item-tag-manage/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { message } from 'antd';
+
+import Action from './action';
+import Filter from './filter';
+import ItemTagManageList from './list';
+import ItemService from '@src/services/Item';
+
+function ItemTagManage() {
+  const [filter, setFilter] = useState({});
+  const [selectedItemIds, setSelectedItemIds] = useState([]);
+
+  const handleChangeFilter = (
+    isReviewedCheck: boolean,
+    hasItemTagsCheck: boolean,
+    itemMinorTypeId?: number,
+    keyword?: string
+  ) => {
+    setFilter({
+      ...(isReviewedCheck ? { isReviewed: isReviewedCheck } : {}),
+      ...(hasItemTagsCheck ? { hasItemTags: hasItemTagsCheck } : {}),
+      ...(itemMinorTypeId ? { itemMinorTypeId } : {}),
+      ...(keyword?.length > 0 ? { keyword } : {}),
+    });
+  };
```

Comment:
```suggestion
  const handleFilterChange = (input : any) => {
    setFilter({
      ...filter,
      ...input
    });
  };
```

1. 함수 네이밍은 영어의 어순을 따라주세요!
2. 위와 같이 parameter를 하나로 줄여보면 좋을 것 같아요

Reply:
1820deb